tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -69,6 +69,53 @@ public function providerInvalidConfigurationElement()
         );
     }
 
+    public function testConfigureResetsExclude()
```

Comment:
We put that kind of tests here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/tests/FixerFactoryTest.php#L380  (not the best place for sure)
More tricky question is how to generalize the test for every fixer without a need to actually provide example configs for every single of them.